src/soundcloud.js

```diff
@@ -0,0 +1,35 @@
+import { URL } from 'url';
+
+export const shouldTransform = string => {
+  return new URL(string).host === 'soundcloud.com';
+};
+
+const getSoundcloudIFrameSrc = string => {
+  return (
+    `https://w.soundcloud.com/player/` +
+    `?url=${encodeURIComponent(string)}` +
```

Comment:
```suggestion
    `?url=${string}` +
```

Reply:
I originally encoded the iframe URL to match Soundcloud's embed feature output. However, it seems to work just fine without encoding the URL so I'm all in for more readable code 😄